music21/chord/__init__.py

```diff
@@ -4770,6 +4770,26 @@ def volume(self, expr):
 
     # --------------------------------------------------------------------------
 
+def sort_pitches(notes_or_pitches, key_pitch):
```

Comment:
naming convention is camel case.  Can you explain why this function is necessary?